### DISCUSSION TITLE
Gif's `align="right"` is not working by default

### DISCUSSION THREAD

**Discussion by imingx** [community/community#17256] [Other Feature Feedback, Questions, & Ideas] [answered]
[Option to prevent animated images from playing automatically](https://github.blog/changelog/2022-05-19-option-to-prevent-animated-images-from-playing-automatically/)

Because of this update↑, the following options were created.

<img src="https://user-images.githubusercontent.com/54981719/170194647-d20a43ec-b9d5-4715-aaa4-aa86eba2fcb5.png" width="40%">

When the user is not logged in or has not set this option, the default is `Sync with system`. But this will cause all gifs to become left-aligned. like:

![image](https://user-images.githubusercontent.com/54981719/170194675-b3e11976-4f4b-4623-bba7-2c1abf3bd041.png)

But actually, the gif has already been set to right-aligned.

```bash
<img src="*.gif" align="right">
```

when I set the options to `Enabled`, it displays properly on the right side:

![image](https://user-images.githubusercontent.com/54981719/170194695-a9a30215-9486-4601-8529-cf900f546e52.png)

**Comment by paveloom** [comment]
Just hit this issue, too. Here's what I have in my [`README.md`](https://github.com/Paveloom/Paveloom):

```
<div align="center">
  <a rel="nofollow" href="https://paveloom.github.io/git/">
    <img width="50%" src="./levitate.gif">
  </a>
</div>
```

And here's the difference between the settings.

Motion > Autoplay animated images > Sync with system:
![sync](https://user-images.githubusercontent.com/49961859/170490260-60ac2920-61ac-4bee-8a65-4fafa1d6d2a0.png)

Motion > Autoplay animated images > Enabled:
![enabled](https://user-images.githubusercontent.com/49961859/170490294-d7d0cb2b-0a90-46b3-bef8-8e4658a1a86e.png)

The link doesn't work in the first option either.

**Comment by RSS1102** [comment]
me too. I think that even if I open it, the visiting users will still view me as `align="left"`. I don't think this will help me.
(It seems that no matter how much I do, it has nothing to do with others.)

**Reply by imingx** [reply]
Yes, it just works for us. 😂

**Comment by vivekweb2013** [comment]
I'm facing this issue too. All the right aligned GIF images from my `README.md` are suddenly appearing to left.

**Reply by RSS1102** [reply]
Maybe there is no good way. This setting can only be "self deception".  You can only set and `README.md` your own according to `sync with system` .If you change the settings, just for your own settings.

**Comment by dipree** [comment, marked as answer]
Both, the problem with `align` and with a percentage unit in the `width` attribute has been fixed.

**Reply by RSS1102** [reply]
ok,I seem to see that the style is restored to the previous style. Thank you for your attention.i try it.

**Reply by paveloom** [reply]
@dipree Thank you. The links on GIFs still don't work, though (see my post above).

**Reply by RSS1102** [reply]
> Thank you. The links on GIFs still don't work, though (see my post above).

I have seen that your home page `Git `is normal.

**Reply by paveloom** [reply]
@RSS1102 Yes, but I mean that there is a link on this GIF, and it still doesn't work with the "Sync with system" option.

**Comment by Nhaifbhuiyan1262** [comment]
> [Option to prevent animated images from playing automatically](https://github.blog/changelog/2022-05-19-option-to-prevent-animated-images-from-playing-automatically/)
> 
> Because of this update↑, the following options were created.
> 
> <img alt="" width="40%" src="https://user-images.githubusercontent.com/54981719/170194647-d20a43ec-b9d5-4715-aaa4-aa86eba2fcb5.png">
> 
> When the user is not logged in or has not set this option, the default is `Sync with system`. But this will cause all gifs to become left-aligned. like:
> 
> ![image](https://user-images.githubusercontent.com/54981719/170194675-b3e11976-4f4b-4623-bba7-2c1abf3bd041.png)
> 
> But actually, the gif has already been set to right-aligned.
> 
> ```shell
> <img src="*.gif" align="right">
> ```
> 
> when I set the options to `Enabled`, it displays properly on the right side:
> 
> ![image](https://user-images.githubusercontent.com/54981719/170194695-a9a30215-9486-4601-8529-cf900f546e52.png)

**Reply by RSS1102** [reply]
If you do this, only you can see that it is "what you think is right".
